Gemfile

```diff
@@ -124,6 +124,7 @@ else
 end
 
 gem 'byebug', '~> 11.1.3'
+gem 'webrick', '~> 1.8.2'
```

Comment:
webrick doesn't have a C extension, so why not do spec.add_dependency in openstudio-gems.gemspec instead?

here:

https://github.com/NREL/openstudio-gems/blob/732fe79d70f26dc3b22f909e4d5b969447734cc5/openstudio-gems.gemspec#L25

Reply:
@kbenne